### ISSUE TITLE
double-edged, quest

### ISSUE THREAD

**Issue by bobeer86** [WarEmu/WarBugs#1348]
quest in chrace, dark elf t1 - after killing the mob you get both parts of the quest progressed, but nowhere to turn in the quest. in original game mob wasn't getting killed and would advance this chain to next quest

**Comment by Reggaewaldo** [comment]
I mad a change in the quest to skip the transition part. After killing the quest you should just turn it in naturally at the quest giver. Made a note in the quest text about this as well.

Closing.

**Comment by dfumagalli** [comment]
For what I recall, the correct quest giver (as also found in online WAR quest database sites) is "Kilraena Dreamlasher", not "Tesmar Dreamstealer" mentioned in the quest text.
By the way, neither of those NPCs will accept the quest for completion, despite I have the two objectives showing as completed.

**Comment by dfumagalli** [comment]
Hello,
As of today, the quest text does not mention the NPC name: "Kilraena Dreamlasher" (I have checked on an online WAR quests database site).
Moreover, she won't take the completed quest in.
Not seeing a name, I have right clicked every NPC I have found at the camp. None of them took the quest.
[Here is a link to a short video I have recorded](https://www.youtube.com/watch?v=EAGKOWqlQFg) that shows the NPC name not being present neither in the tooltip nor the quests tome text, shows the quest having been completed, shows no NPC in the camp taking it.
